tests/dummy/app/templates/application.hbs

```diff
@@ -1,9 +1,35 @@
-{{#materialize-navbar name='ember-cli-materialize'}}
-  <li>{{#link-to 'buttons'}}Buttons{{/link-to}}</li>
-  <li>{{#link-to 'navbar'}}Navbar{{/link-to}}</li>
-  <li>{{#link-to 'cards'}}Cards{{/link-to}}</li>
-  <li>{{#link-to 'collapsible'}}Collapsible{{/link-to}}</li>
-  <li>{{#link-to 'input'}}Input{{/link-to}}</li>
-{{/materialize-navbar}}
+<header>
+  {{#materialize-navbar name='ember-cli-materialize'}}
+    <li>{{#link-to 'buttons'}}Buttons{{/link-to}}</li>
+    <li>{{#link-to 'navbar'}}Navbar{{/link-to}}</li>
+    <li>{{#link-to 'cards'}}Cards{{/link-to}}</li>
+    <li>{{#link-to 'collapsible'}}Collapsible{{/link-to}}</li>
+    <li>{{#link-to 'input'}}Input{{/link-to}}</li>
+  {{/materialize-navbar}}
+</header>
+
+<main>
+  {{outlet}}
+</main>
+
+{{#materialize-footer}}
+  <div class="container">
+    <div class="row">
+      <div class="col l6 s12">
+        <h5 class="white-text">Footer Content</h5>
+        <p class="grey-text text-lighten-4">You can use rows and columns here to organize your footer content.</p>
+      </div>
+      <div class="col l4 offset-l2 s12">
+        <h5 class="white-text">Links</h5>
+        <ul>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 1</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 2</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 3</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 4</a></li>
+        </ul>
+      </div>
+    </div>
+  </div>
```

Comment:
imo this is a pretty huge amount of markup to be passing into this component. We're not saving the developer all that much, compared to the raw materialize markup